paper-dropdown-menu.html

```diff
@@ -242,6 +242,16 @@
           verticalAlign: {
             type: String,
             value: 'top'
+          },
+
+          /**
+           * If you're using PaperInputBehavior to implement your own paper-input-like
+           * element, bind this to the `<input is="iron-input">`'s `readonly` property.
+           */
+          readonly: {
+            type: Boolean,
+            value: false,
+            observer: '_readonlyChanged'
```

Comment:
Thanks for the comment. 
Agreed, this is not going to happen very often (`readonly` property changing while the dropdown is opened), but it should still be the correct behavior, I guess. 
What is the actual weight of adding an observer ?